x-pack/plugins/security_solution/public/common/components/events_viewer/events_viewer.tsx

```diff
@@ -34,6 +34,7 @@ import {
 } from '../../../../../../../src/plugins/data/public';
 import { inputsModel } from '../../store';
 import { useManageTimeline } from '../../../timelines/components/manage_timeline';
+import { showGraphView } from '../../../timelines/components/timeline/body/helpers';
```

Comment:
this was defined here, but perhaps i should move it if im using it this way? looking for guidance from SIEM team.